src/Sessions/logout.php

```diff
@@ -13,9 +12,6 @@ function () {
 			// Expire in 10 minutes.
 			$expiration = time() + ( 10 * 60 );
 			setcookie( LOGOUT_COOKIE_NAME, 'true', $expiration, COOKIEPATH, COOKIE_DOMAIN );
-
-			// Enqueue the script that will perform logout on the client.
-			wp_enqueue_script( SCRIPT_HANDLE_LOGOUT );
```

Comment:
I think this was a neat idea to immediately queue it, for a very quick logout in scenarios where it will work.

Reply:
The script runs on document load, so I don't think it would make a difference, right?

Reply:
Right right, I was thinking of the wp-login.php logout page and I just checked, logout script loads there, so we are good here 👍